src/client/components/pages/Courses/__tests__/tableFields.test.tsx

```diff
@@ -36,27 +36,6 @@ describe('tableFields', function () {
         );
         return getByText(cs50CourseInstance.catalogNumber);
       });
-      it('Should return a component to render a fall level field', function () {
-        const FallEnrollment = retrieveValue('actualEnrollment', TERM.FALL);
-        const { getByText } = render(
-          <FallEnrollment course={cs50CourseInstance} />
-        );
-        return getByText(`${cs50CourseInstance.fall.actualEnrollment}`);
-      });
-      it('Should return a component to render a spring level field', function () {
-        const SpringEnrollment = retrieveValue('actualEnrollment', TERM.SPRING);
-        const { getByText } = render(
-          <SpringEnrollment course={{
-            ...cs50CourseInstance,
-            spring: {
-              ...cs50CourseInstance.spring,
-              actualEnrollment: dummy.int,
```

Comment:
Dropping `dummy.int` here means we can also drop the `dummy` import on lines 15.